bin/upgrade.bash

```diff
@@ -83,6 +83,18 @@ apply() {
     config_validate wc
   fi
 
+  # Create a configmap. This fails if done twice.
+  if [[ "${CK8S_CLUSTER:-}" =~ ^(sc|both)$ ]]; then
+    if ! record_migration_begin sc; then
+      log_fatal "prepare already started in sc (kubectl delete -n kube-system configmap apps-upgrade to try again)"
```

Comment:
```suggestion
      log_fatal "apply already started in sc (kubectl delete -n kube-system configmap apps-upgrade to try again)"
```

Also, would it make sense to add a `ck8s upgrade <prefix> unlock-apply` to do this?
(With some warning+confirm that you probably shouldn't unless you know what you are doing.)

Reply:
I pushed a thing with `--force`, would a separate sub-command be better?

Reply:
I think so, because else some people might just always run with `--force` and that would defeat the purpose.

Reply:
Pushed a `ck8s reset-upgrade` command to bypass some of the argument related logic that `ck8s upgrade ...` does. How's it look?